src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -227,8 +224,6 @@ func (updated *Config) applyFlags(flags map[string]interface{}) error {
 			updated.CloudConfigCA = value.(string)
 		case "frontedconfig":
 			updated.FrontedCloudConfig = value.(string)
-		case "instanceid":
```

Comment:
Because it set the device ID in the ClientConfig, which doesn't exist anymore. That's the one thing we lose with this change -- the server can no longer set the device ID. My thought was to add that if we ever need it @xiam.